src/components/TransactionList.js

```diff
@@ -24,41 +30,39 @@ const TransactionList = (props) => {
       }
       <div className='transaction-list'>
         {transactions.map((transaction, idx) => {
+          let template = transaction.template;
+
           let categoryName;
-          if (transaction.template.categoryId) {
-            const category = categories.find((category) => category.id === transaction.template.categoryId);
+          if (template.categoryId) {
+            const category = categories.find((category) => category.id === template.categoryId);
             if (category) {
               categoryName = category.displayName;
             }
           }
           return (
             <div key={idx} className='transaction'>
               <button className='delete-transaction-button' onClick={()=>removeScheduledTransaction(transaction.id)}>❌</button>
+              
               <div className='transaction-details'>
                 <div className='transaction-left'>
-                  {transaction.template.amount < 0 ?
-                    <div className='expense amount'>
-                      ${-transaction.template.amount}
-                    </div> :
-                    <div className='income amount'>
-                      ${transaction.template.amount}
+                  <div className='memo'>{template.memo}</div>
+                  { categoryName &&
+                    <div className='category'>
+                      Category: {categoryName}
                     </div>
                   }
-                  <div className='account'>
-                    Account: {transaction.template.account}
-                  </div>
                 </div>
                 <div className='transaction-right'>
-                  <div className='memo'>
-                    {transaction.template.memo}
+                  <div className={`${getAmountClass(transaction.template)} amount`}>
+                    { template.type === TransactionType.Transfer && <span>⇆ </span> }
+                    ${template.amount.toLocaleString('en-US')}
+                  </div>
+                  <div className='account'>
+                    Account: {template.targetAccount}
```

Comment:
Maybe if it is a transfer, we should show the targetAccount and the originAccount?

Reply:
You're right, I totally forgot to go back and add that. Thanks